klippy/extras/palette2.py

```diff
@@ -618,6 +637,13 @@ def _run_Smart_Load(self, eventtime):
             return self.reactor.NOW
         return eventtime + AUTOLOAD_TIMER
 
+    def get_status(self, eventtime=None):
+        status = {
+            "ping": self.omega_pings[-1],
+            "remaining_load_length": self.remaining_load_length,
+            "is_splicing": self.is_splicing
+        }
+        return status
```

Comment:
When adding a `get_status()` method, please also update the docs in docs/Command_Template.md .